src/practices/LanguageIndependent/LicenseIsPresentPractice.ts

```diff
@@ -37,14 +47,32 @@ export class LicenseIsPresentPractice implements IPractice {
     return PracticeEvaluationResult.notPracticing;
   }
 
-  async fix() {
-    await cli.action.pauseAsync(
-      () =>
-        new Promise((resolve) => {
-          env.run('license', () => {
-            resolve();
-          });
-        }),
-    );
+  private getDefaultLicense(ctx: FixerContext) {
+    if (ctx.config && (ctx.config as PracticeOverride).override?.defaultLicense) {
+      return (ctx.config as PracticeOverride).override.defaultLicense;
+    } else if (ctx.scanningStrategy.accessType === AccessType.private) {
+      return 'UNLICENSED';
```

Comment:
This is maybe too harsh. We might prefer some more permissive license even for "private for now" repositories.

Reply:
I don't know. We could just not add a license? => https://softwareengineering.stackexchange.com/a/312011

Reply:
That's basically what `UNLICENSED` option does: https://github.com/jozefizso/generator-license/blob/master/app/templates/UNLICENSED.txt

So I guess, we can keep it that way.

Reply:
Ok, let's have an UNLICENSED. The community can update the license later. :)